IRremoteESP8266.cpp

```diff
@@ -470,7 +470,7 @@ void ICACHE_FLASH_ATTR IRsend::sendRCMM(uint32_t data, uint8_t nbits) {
 void ICACHE_FLASH_ATTR IRsend::sendPanasonic64(unsigned long long data,
                                                unsigned int nbits,
                                                unsigned int repeat) {
-  enableIROut(37);  // Set IR carrier frequency
+  enableIROut(36700U);  // Set IR carrier frequency of 36.7kHz.
   IRtimer usecTimer = IRtimer();
 
   for (uint16_t i = 0; i <= repeat; i++) {
```

Comment:
I'm not sure why you think that. Can you explain why? 'Cause I'm missing it. I think that loop is what allows a caller to specify how many times they want the message repeated. If it doesn't, then it isn't doing what I intended, and needs to be fixed.
e.g. This is what I think would happen: A call of `sendPanasonic64(0xF00BAA, 48, 2);` would produce 3 x Panasonic64 0xF00BAA  messages (1 message + 2 repeats)